server/server_comm/test_runner/views.py

```diff
@@ -109,8 +111,16 @@ def run_node(self, node):
         try:
             result = None
             if node.node_type == Node.ASSERT:
-                # TODO add code from LIL-91
-                result = True
+                node_checker.check_valid_input(node.function)
+                try:
+                    result = exec(node.function)
+                    if result:
+                        result = "Assert OK"
+                    else:
+                        result = "Assert failed"
+                except Exception as e:
```

Comment:
I think you can omit this try/catch block since the whole function is wrapped in one. We'd like those errors to be sent to the user in case they write invalid python code instead of just catching them and continuing